3rdparty/compiler-rt/compiler-rt/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2011-2020 Intel Corporation. All rights reserved.
```

Comment:
Is this file actually used? The CMakeLists.txt file looks like it doesn't depend on this at all.